app/views/admin/claims/show.html.erb

```diff
@@ -4,12 +4,18 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render("shared/error_summary", instance: @decision, errored_field_id_overrides: { "result": "decision_result_approved" }) if @decision.errors.any? %>
 
-    <% if @claim.payroll_gender_missing? %>
+    <% if @claim.payroll_gender_missing? and not @claim.pii_removed? %>
```

Comment:
I find it a bit more "Rubyish" to use a `!` rather than `not` when checking the reverse:

```suggestion
    <% if @claim.payroll_gender_missing? and !@claim.pii_removed? %>
```

Reply:
Note that this is using the control-flow `and` and not the logical operator `&&` here. Stick with the logical operators `&&` and `||` in conditional statements. See http://www.virtuouscode.com/2010/08/02/using-and-and-or-in-ruby/ for more info on the different. TBH I'm surprised that our Ruby standard checks allowed this, I thought they disallowed them...